tests/integration/targets/openssh_keypair/tasks/setup_bcrypt.yml

```diff
@@ -0,0 +1,24 @@
+---
```

Comment:
This should be at some point moved to tests/integration/targets/setup_bcrypt/ so it can be used by other integration tests as well. No need to do that in this PR though :)

Reply:
Yes, sorry was just pushing through issues with CI so there's definitely cleanup work to be done.